components/time-picker/index.tsx

```diff
@@ -20,10 +21,11 @@ const RangePicker = React.forwardRef<any, TimeRangePickerProps>((props, ref) =>
 
 export interface TimePickerProps extends Omit<PickerTimeProps<Moment>, 'picker'> {
   addon?: () => React.ReactNode;
+  popupClassName?: string;
 }
 
 const TimePicker = React.forwardRef<any, TimePickerProps>(
-  ({ addon, renderExtraFooter, ...restProps }, ref) => {
+  ({ addon, renderExtraFooter, popupClassName, dropdownClassName, ...restProps }, ref) => {
```

Comment:
Why do you also need the `dropdownClassName` prop?

Reply:
Well in `3.x`,  `rc-time-picker` was being using which had `popupClassName` prop to add class to the popup.
But in `4.x`, `rc-picker` is being used which does not have `popupClassName`.
Instead it has `dropdownClassName`

Reply:
`TimePickerProps` extends `RCPickerTimeProps`.
So one might use `dropdownClassName` instead of `popupClassName`.
So i have combined both classes.

Reply:
`dropdownClassName` has never been disclosed in the document, so please use `popupClassName` instead, that is enough.